README.md

```diff
@@ -232,6 +234,9 @@ See [Log Level](https://github.com/hms-networks/sc-flexy-logger-lib#log-level) f
         }
     }
 ```
+#### AutoRestart
```

Comment:
This should probably say what the values mean, "Setting the AutoRestart configuration parameter to true..."